tests/NHSD.BuyingCatalogue.Organisations.Api.IntegrationTests/Steps/OdsOrganisationSteps.cs

```diff
@@ -39,6 +42,21 @@ public async Task GivenOrganisationsExist(Table table)
                 var odsApiOrganisation = TransformIntoOdsApiFormat(org);
                 var odsApiOrganisationAsJson = JsonConvert.SerializeObject(odsApiOrganisation);
                 await api.SetUpGetEndpoint(org.OdsCode, odsApiOrganisationAsJson);
+
+                var organisation = OrganisationEntityBuilder
+                    .Create()
+                    .WithName(org.Name)
+                    .WithOdsCode(org.OdsCode)
+                    .WithAddressLine1(org.AddressLine1)
+                    .WithAddressLine2(org.AddressLine2)
+                    .WithAddressLine3(org.AddressLine3)
+                    .WithAddressLine4(org.AddressLine4)
+                    .WithAddressTown(org.Town)
+                    .WithAddressCounty(org.County)
+                    .WithAddressPostcode(org.PostCode)
+                    .WithAddressCountry(org.Country)
+                    .Build();
```

Comment:
We should also set the ID so that we can confirm that the expected ID is returned as part of the integration tests.

Reply:
The setting of ID was failing for `AlsoBuyer` value due to data truncation. 
As I was not aware of the reasoning behind usage of this ID value which is also mentioned in `docker-compose.yml`, the addition of ID was removed.